### PR TITLE
IWTomics bug fix

### PR DESCRIPTION
**Pull request by fabio-cumbo**
Fixed a bug about optional input fields.
If a value has not been selected in an optional input field, do not send it as parameter to the tool executable.

### CODE REVIEW

File: tools/iwtomics/loadandplot.xml [hunk 1]
```diff
@@ -15,8 +15,8 @@
       regionsgalaxyids='c(${region_galaxyids})'
       #set region_names = '|'.join( [ str( $r.name ) for $r in $regions ] )
       regionsfilenames='${region_names}'
-      #set region_header = $regionsheader
-      #if $region_header != "":
+      #set region_header = str( $regionsheader )
+      #if $region_header != "None":
         regionsheaderfile='${region_header}'
```
Comment (nsoranzo) [inline tools/iwtomics/loadandplot.xml]: I think you can simplify lines 18-20 to:
```
      #if $regionsheader:
         regionsheaderfile='${regionsheader}'
```

File: tools/iwtomics/loadandplot.xml [hunk 2]
```diff
@@ -26,8 +26,8 @@
       featuresgalaxyids='c(${feature_galaxyids})'
       #set feature_names = '|'.join( [ str( $f.name ) for $f in $features ] )
       featuresfilenames='${feature_names}'
-      #set feature_header = $featuresheader
-      #if $feature_header != "":
+      #set feature_header = str( $featuresheader )
+      #if $feature_header != "None":
         featuresheaderfile='${feature_header}'
```
Comment (nsoranzo) [inline tools/iwtomics/loadandplot.xml]: Similarly here.

**Comment by fabio-cumbo** [comment]
Done! thanks @nsoranzo

File: tools/iwtomics/loadandplot.xml [hunk 3]
```diff
@@ -15,8 +15,7 @@
       regionsgalaxyids='c(${region_galaxyids})'
       #set region_names = '|'.join( [ str( $r.name ) for $r in $regions ] )
       regionsfilenames='${region_names}'
-      #set region_header = $regionsheader
-      #if $region_header != "":
+      #if $regionsheader:
         regionsheaderfile='${region_header}'
```
Comment (nsoranzo) [inline tools/iwtomics/loadandplot.xml]: @fabio-cumbo You need to update also this line changing `region_header` to `regionsheader`

File: tools/iwtomics/loadandplot.xml [hunk 4]
```diff
@@ -26,8 +25,7 @@
       featuresgalaxyids='c(${feature_galaxyids})'
       #set feature_names = '|'.join( [ str( $f.name ) for $f in $features ] )
       featuresfilenames='${feature_names}'
-      #set feature_header = $featuresheader
-      #if $feature_header != "":
+      #if $featuresheader:
         featuresheaderfile='${feature_header}'
```
Comment (nsoranzo) [inline tools/iwtomics/loadandplot.xml]: Same here

**Comment by fabio-cumbo** [comment]
my mistake, thanks!

**Comment by nsoranzo** [comment]
Thanks @fabio-cumbo!

**Pull request merged**